.gitignore

```diff
@@ -10,3 +10,129 @@ build/**
 **/*.nef
 **/*.rw2
 env/**
+
```

Comment:
Are most of these standard excludes or related to packaging for pip? Was just curious - no change needed.

Reply:
These are standard includes for python. The python compiler will add files to the repo which will get checked into git unless they're excluded by .gitignore